brainkeeper/src/app/services/image-service.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ImageServiceService {
```

Comment:
Please rename to ImageService, also rename file to image.service.ts

Reply:
Done.